src/common/helpers/tracking/tracking.ts

```diff
@@ -0,0 +1,86 @@
+import { createInstance } from '@datapunt/matomo-tracker-react';
```

Comment:
This file is very Matomo specific, so should it be named `matomo.ts` or `matomoTracking.ts` or something like that?

Reply:
I named it `tracking`,  because that's what it does and if tracking service is changed, changes can be done in the same file and filename can be the same. :)

Reply:
Hmm, that makes sense too; Matomo is an internal implementation detail.

Reply:
Then again, Matomo isn't really hidden within this module. `useTrackingInstance()` returns a `MatomoTracker` which is to be used with a `MatomoProvider`, so this is quite Matomo specific looking from the outside also.

Reply:
Changed to matomoTracking